exporter/exporter.go

```diff
@@ -166,6 +171,16 @@ func (e *Exporter) makeRegistry(ctx context.Context, client *mongo.Client, topol
 		e.opts.EnableIndexStats = true
 	}
 
+	// arbiter only have isMaster privileges
+	if isArbiter {
+		e.opts.EnableDiagnosticData = false
+		e.opts.EnableDBStats = false
+		e.opts.EnableCollStats = false
+		e.opts.EnableTopMetrics = false
+		e.opts.EnableReplicasetStatus = false
+		e.opts.EnableIndexStats = false
```

Comment:
Is it related to the issue we are solving in this PR?
it would be good to write some logs here to show that we are disabling collectors.

Reply:
it is a separate issue.
if we run the exporter to monitoring arbiter role instance. the exporter will be hung to stuck.  
It seems I should make a separate issue and pr.  if possible, i will learn how to separate commits from pr before this week ends

Reply:
Great, thank you

Reply:
Do you want to expose URL in mongodb_exporter logs/metrics or somehow else?

Reply:
> Do you want to expose URL in mongodb_exporter logs/metrics or somehow else?

I want to expose the url in cmdline, which can be easily identified by `ps` command, the log or metric need more works to fetch url info. hard to manipulate exporter instance

Reply:
aha, ok, it makes sense